newton/_src/sensors/warp_raytrace/utils.py

```
@@ -145,14 +145,151 @@ def convert_ray_depth_to_forward_depth_kernel(
+        out[n, y, x, 3] = wp.uint8(255)
+        return
+    i = wp.int32(idx)
+    out[n, y, x, 0] = colors[i, 0]
+    out[n, y, x, 1] = colors[i, 1]
+    out[n, y, x, 2] = colors[i, 2]
+    out[n, y, x, 3] = wp.uint8(255)
+
+
+def _validate_batched_rgba_out_buffer(
+    name: str,
+    out_buffer: wp.array[Any],
```

Ruff (0.15.11)

[error] Undefined name `Any`

(F821)